packages/terra-open-source-scripts/src/terra-cli/release/releaseMonoRepoHandler.js

```diff
@@ -0,0 +1,37 @@
+const childProcess = require('child_process');
```

Comment:
nit: rename file to `release-monorepo` and rename the non-mono-release file to `release`